api/src/utils/github.js

```diff
@@ -58,11 +58,37 @@ export const getUserByName = async (username) => {
   log.debug(`getUserByName ${username}`)
   const installations = await getAuthenticatedApps()
   const config = getConfig()
-
   const response = await installations.apps[
-    config.primaryOrg
+    config.primaryOrg.toLowerCase()
   ].authenticatedRequest('GET /users/{username}', {
     username,
   })
   return response.data
 }
+
+export const getRequestStatuses = async (requests) => {
+  log.info(`getRequestStatuses`)
+  let pendingUserRequests = []
+  const installations = await getAuthenticatedApps()
+
+  const config = getConfig()
+
+  for (const org of config.orgs) {
+    log.debug(`org being searched ${org}`)
```

Comment:
this debug message should describe what you are searching for. In this case its not so much a search but a fetching invitations for an org